Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANSILLParameterScan.py

```diff
@@ -0,0 +1,277 @@
+                                                  action=FileAction.OptionalLoad,
+                                                  extensions=['nxs']),
+                             doc='Absorber (Cd/B4C) run(s).')
+
+        self.declareProperty(MultipleFileProperty('ContainerRuns',
+                                                  action=FileAction.OptionalLoad,
+                                                  extensions=['nxs']),
+                             doc='Empty container run(s).')
+
+        self.setPropertyGroup('SampleRuns', 'Numors')
+        self.setPropertyGroup('AbsorberRuns', 'Numors')
+        self.setPropertyGroup('ContainerRuns', 'Numors')
+
+        self.declareProperty(MultipleFileProperty('SensitivityMaps',
+                                                  action=FileAction.OptionalLoad,
+                                                  extensions=['nxs']),
+                             doc='File containing the map of relative detector efficiencies.')
+
+        self.declareProperty(MatrixWorkspaceProperty('SensitivityOutputWorkspace', '',
+                                                     direction=Direction.Output,
+                                                     optional=PropertyMode.Optional),
+                             doc='The output sensitivity map workspace.')
+
+        self.declareProperty(FileProperty('DefaultMaskFile', '', action=FileAction.OptionalLoad, extensions=['nxs']),
+                             doc='File containing the default mask to be applied to all the detector configurations.')
+
+        self.declareProperty(MultipleFileProperty('MaskFiles',
+                                                  action=FileAction.OptionalLoad,
+                                                  extensions=['nxs']),
+                             doc='File(s) containing the beam stop and other detector mask.')
+
+        self.copyProperties('SANSILLReduction', ['NormaliseBy'])
+
+        self.declareProperty('SampleThickness', 0.1, validator=FloatBoundedValidator(lower=0.),
+                             doc='Sample thickness [cm]')
+
+        self.declareProperty('Observable', 'Omega.value',
+                             doc='Parameter from the sample logs along which the scan is made')
+
+        self.declareProperty('PixelYMin', 0, validator=IntBoundedValidator(lower=0),
+                             doc='Minimal y-index taken in the integration')
+        self.declareProperty('PixelYMax', 320, validator=IntBoundedValidator(lower=0),
+                             doc='Maximal y-index taken in the integration')
+
+        self.setPropertyGroup('SensitivityMaps', 'Options')
+        self.setPropertyGroup('SensitivityOutputWorkspace', 'Options')
+        self.setPropertyGroup('DefaultMaskFile', 'Options')
+        self.setPropertyGroup('MaskFiles', 'Options')
+        self.setPropertyGroup('NormaliseBy', 'Options')
+        self.setPropertyGroup('SampleThickness', 'Options')
```

Comment:
Not sure about the thickness; sample is rotating, so thickness is different for each omega.